src/SymfonyCache/CustomTtlListener.php

```diff
@@ -41,13 +46,15 @@ class CustomTtlListener implements EventSubscriberInterface
     public const SMAXAGE_BACKUP = 'FOS-Smaxage-Backup';
 
     /**
-     * @param string $ttlHeader     The header name that is used to specify the time to live
-     * @param bool   $keepTtlHeader Keep the custom TTL header on the response for later usage (e.g. debugging)
+     * @param string $ttlHeader       The header name that is used to specify the time to live
+     * @param bool   $keepTtlHeader   Keep the custom TTL header on the response for later usage (e.g. debugging)
+     * @param bool   $disableFallback If the custom TTL header is not set, do not fall back to s-maxage
      */
-    public function __construct($ttlHeader = 'X-Reverse-Proxy-TTL', $keepTtlHeader = false)
+    public function __construct($ttlHeader = 'X-Reverse-Proxy-TTL', $keepTtlHeader = false, $disableFallback = false)
```

Comment:
we should have used an `$options` array. this becomes a bit cumbersome with several options. then again, from the bundle it works just fine.

```suggestion
    public function __construct($ttlHeader = 'X-Reverse-Proxy-TTL', $keepTtlHeader = false, $fallbackToSmaxage = true)
```